pkgs/top-level/linux-kernels.nix

```diff
@@ -532,11 +532,11 @@ in {
 
     system76-acpi = callPackage ../os-specific/linux/system76-acpi { };
 
-    system76-power = callPackage ../os-specific/linux/system76-power { };
+    system76-power = lib.warn "kernelPackages.system76-power is now pkgs.system76-power" (callPackage ../os-specific/linux/system76-power { });
```

Comment:
Move these down to the section at line 604 that is conditional on `config.allowAliases` 
 to fix eval cleanliness and use `pkgs.system76-power` instead of `(callPackage ...)`

Reply:
Ack